modules/compliance-tailored-profiles.adoc

```diff
@@ -74,15 +72,24 @@ spec:
 |`setValues`
 | A list of name, rationale, and value groupings. Each name refers to a name of the value set. The rationale is human-readable text describing the set. The value is the actual setting.
 |===
-+
-. Add the profile to the `ScanSettingsBinding` object:
+
+.. Create the TailoredProfile object:
 +
 [source,terminal]
 ----
-$ cat nist-moderate-modified.yaml
+$ oc create -n compliance.openshift -f new-profile-node.yaml <1>
 ----
+<1> The TailoredProfile object is created in the default compliance.openshift namespace.
```

Comment:
I think the CO installs using the openshift-compliance namespace. That's also the namespace I see in other sections of this documentation. Do we want to use there here, too?

Reply:
Oh - looks like I flipped it. Good catch.